cooltools/cli/coverage.py

```diff
@@ -67,8 +70,9 @@
     default=1,
     type=int,
 )
+@pool_decorator
```

Comment:
I wonder if the pool_decorator should instead be applied to the api version of this function? cc @Phlya